firefly/app.py

```diff
@@ -25,7 +26,7 @@
 ctx.request = None
 
 class Firefly(object):
-    def __init__(self, auth_token=None, allowed_origins=""):
+    def __init__(self, name, auth_token=None, allowed_origins="", flask_app=None):
```

Comment:
use `name=None` and set it to `"firefly"` if not specified. That way you don't have to change any of the existing code.